scripts/pages/cart.js

```diff
@@ -16,6 +15,19 @@
                     return false;
                 }
             });
+
+
+            var visaCheckoutSettings = HyprLiveContext.locals.siteContext.checkoutSettings.visaCheckout;
+            var pageContext = require.mozuData('pagecontext');
+            if (visaCheckoutSettings.isEnabled) {
+                window.onVisaCheckoutReady = initVisaCheckout;
```

Comment:
The Visa Checkout guidelines ask for an `onVisaCheckoutReady` global function to be added to the document `<head>` tag. We have assumed that their intent is to make sure that this function is defined before the Visa Checkout SDK is loaded. Our code guarantees this a different way: the RequireJS library governs the order in which our modules load, and this line defined the global `onVisaCheckoutReady` before running `require()` to remote-load the Visa Checkout SDK on the following line.